KV cloud tutorial and updated links n labels conceptual article (#1158)
fermyon/developer: content/spin/v2/key-value-store-tutorial.md

<!-- @selectiveCpy -->

```bash
$ spin cloud deploy
spin deploy
Uploading spin-key-value version 0.1.0-r234fe5a4 to Fermyon Cloud...
Deploying...
App "spin-key-value" accesses a key value store labeled "default"
Would you like to link an existing key value store or create a new key value store?:
> Use an existing key value store and link app to it
  Create a new key value store and link the app to it
```

>> If you're interested in learning more about how to link your Spin app to different key value store instances on Fermyon Cloud, chek out our Key Value Links and Labels tutorial. 

Congratulations, you have a Spin application and associated key value store running up in Fermyon Cloud! You can visit it by clicking on the Spin application's domain name generated in the CLI output, which has the following pattern: `spin-key-value-<RANDOM>.fermyon.app`. 

## Next Steps

* Explore the contents of your Key Value store with the [Key Value Store Explorer template](https://developer.fermyon.com/hub/preview/template_kv_explorer)

> * Learn about linking your applications to different [Key Value Stores on Fermyon Cloud](TODO: link tutorial) 

Reviewer: ```suggestion
* Learn about linking your applications to different [Key Value Stores on Fermyon Cloud](../../cloud/kv-cloud-tutorial.md) 
```

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```
